lfs/pointer.go

```diff
@@ -22,7 +22,7 @@ var (
 	}
 	latest      = "https://git-lfs.github.com/spec/v1"
 	oidType     = "sha256"
-	oidRE       = regexp.MustCompile(`\A[[:alnum:]]{64}`)
+	oidRE       = regexp.MustCompile(`\A[0-9A-Fa-f]{64}\z`)
```

Comment:
https://github.com/git-lfs/git-lfs/blob/master/docs/spec.md#the-pointer says this is lowercase:

`oid tracks the unique object id for the file, prefixed by its hashing method: {hash-method}:{hash}. Currently, only sha256 is supported. The hash is lower case hexadecimal.`

Do we care about supporting uppercase?